api/build.gradle

```diff
@@ -160,6 +160,30 @@ task generateMoodleClient {
   }
 }
 
+def rdrApiFile = 'src/main/resources/rdr.yaml'
```

Comment:
Do we need to keep their swagger file locally, or can we pull it in from a URL so we always have the latest?